### PR TITLE
Defining and Enabling /api/v1.0 of the API

### PR DESCRIPTION
**Pull request by abellotti**
Enabling /api/v1.0
- OpenAPI 3.0.0  for v1.0
- Controllers (based on v0.1)
- Specs added for /api/v1.0

### CODE REVIEW

File: app/controllers/api/v1x0.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
+module Api
+  module V1x0
+    class RootController < ApplicationController
+      def openapi
+        render :json => Api::Docs["1.0"].content.to_json
+      end
+    end
+
+    class AuthenticationsController < Api::V0x0::AuthenticationsController
+      include Api::V0x1::Mixins::IndexMixin
```
Comment (agrare) [inline app/controllers/api/v1x0.rb]: @abellotti could we also just drop `v0.0` and move the `v0x1` index_mixin into the base mixins?

Comment (abellotti) [reply app/controllers/api/v1x0.rb]: no one is using /api/v0.0  (just the /internal/v0.0), should I drop that in this PR ?

Comment (abellotti) [reply app/controllers/api/v1x0.rb]: ok, /api/v0.0 goes bye bye

**Comment by abellotti** [comment]
ping @gtanzillo will take some work to remove /api/v0.0, let's do this in a followup-PR.

**Pull request merged**